aviary/docs/user_guide/propulsion.ipynb

```diff
@@ -340,7 +340,7 @@
    },
    "outputs": [],
    "source": [
-    "# ********NOTE********\n",
+    "# ****NOTE****\n",
     "# this fails if no engine models are provided:\n",
```

Comment:
@jkirk5

Reply:
so for this pr, just remove these two sections:
![image](https://github.com/user-attachments/assets/2d233778-c992-451f-8a3e-85f33dbabfc2)

Reply:
Yeah that is ok to remove (for now), it needs to be replaced with an entirely new section for the engine deck builder